test/RelayHubGasCalculations.test.ts

```diff
@@ -337,6 +345,88 @@ contract('RelayHub gas calculations', function ([_, relayOwner, relayWorker, rel
       })
   })
 
+  describe('relayCall()\'s msg.data cost calculations', function () {
+    enum RelayCallDynamicArg {
+      APPROVAL_DATA = 'approvalData',
```

Comment:
giving string aliases to these enums does not seem useful

Reply:
fixed